robots.txt

```diff
@@ -1,8 +1,4 @@
 User-agent: *
-Disallow: /addons/
-Disallow: /plugins/
 Disallow: /app/
-Disallow: /media/
```

Comment:
I'm not confident enough with the content of every folder. I know that I have an issue with the folder media so feel free to update my PR.